### ISSUE TITLE
acceptance test code difference between stable10 & master

### ISSUE THREAD

**Issue by individual-it** [owncloud/core#33499]
`bootstrap/Sharing.php` is different in master & stable10 
code added in https://github.com/owncloud/core/pull/22750 is missing in master
```
				if (\substr($field, 0, 10) === "share_with") {
					$value = \str_replace(
						"REMOTE",
						$this->getRemoteBaseUrl(),
						$value
					);
					$value = \str_replace(
						"LOCAL",
						$this->getLocalBaseUrl(),
						$value
					);
				}
				if (\substr($field, 0, 6) === "remote") {
					$value = \str_replace(
						"REMOTE",
						$this->getRemoteBaseUrl() . '/',
						$value
					);
					$value = \str_replace(
						"LOCAL",
						$this->getLocalBaseUrl() . '/',
						$value
					);

**Comment by ownclouders** [comment]
[GitMate.io](https://gitmate.io) thinks the contributor most likely able to help you is @PVince81.

Possibly related issues are https://github.com/owncloud/core/issues/31579 (test), https://github.com/owncloud/core/issues/22485 (test), https://github.com/owncloud/core/issues/22492 (test), https://github.com/owncloud/core/pull/31517 ([stable10] Acceptance test coding standard changes), and https://github.com/owncloud/core/pull/32892 ([stable10] Make targets for acceptance tests).

**Comment by phil-davis** [comment]
I added this to the sprint - we should resolve this sort of crud "as soon as we notice it"

**Comment by individual-it** [comment]
fixed by #33501
